app/services/platform.rb

```diff
@@ -4,9 +4,21 @@
 
 # Methods related to connecting to other platform services
 module Platform
+  RETRY_OPTIONS = {
+    max: 3,
+    interval: 0.05,
+    interval_randomness: 0.5,
+    backoff_factor: 2,
+    methods: %i[get],
+    exceptions: [
+      *Faraday::Request::Retry::DEFAULT_EXCEPTIONS, Faraday::ConnectionFailed
```

Comment:
I've changed this now to use the DEFAULT_EXCEPTIONS, which include calls that return `RetryAfter`, Timeout, etc.. It seems cleaner than writing out all of the possibilities ourselves